socorro/external/crashstorage_base.py

```diff
@@ -106,7 +106,15 @@ def __init__(self):
         # implementation. This may be fetched by a client of the crashstorge so that it
         # can determine if it can try a failed storage operation again.
         self.exceptions_eligible_for_retry = ()
+
+        # Configure logger
         self.logger = logging.getLogger(__name__ + "." + self.__class__.__name__)
+        handler = logging.StreamHandler()
+        formatter = logging.Formatter(
+            "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+        )
+        handler.setFormatter(formatter)
+        self.logger.addHandler(handler)
```

Comment:
This was previously using a logger that inherited from `RootLogger` that had no handlers. I had to add logger config to get it to output at all.